rpcs3/Emu/Memory/vm.cpp

```diff
@@ -783,6 +783,7 @@ namespace vm
 			{
 				std::make_shared<block_t>(0x00010000, 0x1FFF0000), // main
 				std::make_shared<block_t>(0x20000000, 0x10000000), // user
+				std::make_shared<block_t>(0x40000000, 0x30000000), // rsx contexts
```

Comment:
lv1/2 allows 3 rsx contexts i think, with vsh normally at 0x60000000, game context is at 0x40000000,  so i assume it gives the full 0x10000000 for each context. Why end of list?